src/ontology/mondo-ingest.Makefile

```diff
@@ -159,6 +159,17 @@ $(COMPONENTSDIR)/icd10who.owl: $(TMPDIR)/icd10who_relevant_signature.txt | compo
 		remove -T config/properties.txt --select complement --select properties --trim true \
 		annotate --ontology-iri $(URIBASE)/mondo/sources/icd10who.owl --version-iri $(URIBASE)/mondo/sources/$(TODAY)/icd10who.owl -o $@; fi
 
+$(COMPONENTSDIR)/icd11foundation.owl: $(TMPDIR)/icd11foundation_relevant_signature.txt | component-download-icd11foundation.owl
```

Comment:
## `build-mondo-ingest` _@finally_
- [ ] Run `build-mondo-ingest` and considered passing if [lexmatch table](https://github.com/monarch-initiative/mondo-ingest/blob/main/src/ontology/lexmatch/unmapped_doid_lex_exact.tsv) has more than 1k rows. @joeflack4

Reply:
Been running for 3 hours so far. I'll check on it later tonight or tomorrow.

Reply:
Ok, it just finished. Unfortunately there were no `lexmatch/unmapped_icd11foundation*.txv` generated. I'll look into it this week, hopefully tomorrow. I am supposing there is some configuration missing somewhere. I wonder if there might be a hardcoding of ontology names in the lexmatch scripts.

Reply:
> Fix issues where lexmatch table is not getting generated

The `components` dir does not seem to have any `icd11.owl` file , that may be why there are no lexmatch files for icd11

Reply:
Ah yeah, it should be `icd11foundation`. We changed the name.